src/ptvsd/_vendored/pydevd/_pydevd_bundle/pydevd_net_command_factory_xml.py

```diff
@@ -152,6 +152,30 @@ def make_thread_killed_message(self, id):
         except:
             return self.make_error_message(0, get_exception_traceback_str())
 
+    def _iter_visible_frames_info(self, py_db, frame, frame_id_to_lineno):
+        while frame is not None:
+            if frame.f_code is None:
+                continue  # IronPython sometimes does not have it!
+
+            method_name = frame.f_code.co_name  # method name (if in method) or ? if global
+            if method_name is None:
+                continue  # IronPython sometimes does not have it!
+
+            abs_path_real_path_and_base = get_abs_path_real_path_and_base_from_frame(frame)
+            if py_db.get_file_type(abs_path_real_path_and_base) == py_db.PYDEV_FILE:
```

Comment:
This should also skip ptvsd files. is there a way to configure that?

Reply:
`ptvsd` files are already handled there (done at: https://github.com/Microsoft/ptvsd/blob/master/src/ptvsd/wrapper.py#L183)

Reply:
Ah! right.